chart/gatekeeper-operator/helm-modifications/helm-modifications.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: gatekeeper-controller-manager-service
+  namespace: gatekeeper-system
+spec:
+  selector:
+    app: GATEKEEPER_APP_LABEL
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: configs.config.gatekeeper.sh
+  annotations:
+    helm.sh/hook: crd-install
```

Comment:
`crd-install` hook is not supported in Helm 3. Can we use the workaround at https://github.com/helm/helm/pull/6243?

Reply:
Actual stable version of helm is Helm 2, and it will be so long time (until helm 3 would be stable).
Until Helm 3 released we still need helm chart.

Please, please, please, keep this hook.

Reply:
Agreed, for this reason I prefer to leave this as is and specify that this is for Helm v2, and leave v3 for another day.